crates/rolldown/tests/esbuild/default/require_fs_node_minify/_config.json

```diff
@@ -5,7 +5,9 @@
         "name": "entry",
         "import": "entry.js"
       }
-    ]
+    ],
+    "format": "cjs",
```

Comment:
update the configuration, align with esbuild https://github.com/evanw/esbuild/blob/d34e79e2a998c21bb71d57b92b0017ca11756912/internal/bundler_tests/bundler_default_test.go#L1457-L1473